sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -239,10 +240,18 @@ public partial class ServiceBusProcessor : System.IAsyncDisposable
         public override string ToString() { throw null; }
         public void UpdateConcurrency(int maxConcurrentCalls) { }
     }
+    public partial class ServiceBusProcessorDiagnostics
+    {
+        protected internal ServiceBusProcessorDiagnostics() { }
+        public System.DateTimeOffset? LastReceiveAttemptedTime { get { throw null; } }
+        public System.DateTimeOffset? LastReceiveSucceededTime { get { throw null; } }
```

Comment:
LastReceivedMessageTime

Reply:
Potentially not have the succeeded time since the user would know this information from their handler.